content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -1010,9 +994,72 @@ Support for [`autoincrement()`](../../../reference/api-reference/prisma-schema-r
 
 **Relational database connectors** implement `autoincrement()`, `dbgenerated()`, and `now()` at database level. The **MongoDB connector** does not support `autoincrement()`, and `now()` is implemented at Prisma level. `dbgenerated()` can only be used to generate an `ObjectId`.
 
-## Relations
+## Constraints and indexes in the Prisma schema
```

Comment:
This headline is weird as it says "in the Prisma Schema" although we are in a "Prisma schema" docs page. 
Without it, it is "too random" though as well.

Reply:
Unique and Index are both attributes, maybe this is more about "Naming Attributes"? 
Update: Well, except for Relations which are its own thing of course and destroy this idea :(

Reply:
Another thought: This section somehow feels like it belong into a "SQL concepts in Prisma schema" page or section.

Reply:
As per discussion - there is now a 'naming stuff' page under Prisma schema